### DISCUSSION TITLE
Error: ServerFunctionsProvider requires a serverFunction prop

### DISCUSSION THREAD

**Discussion by mattiaz9** [payloadcms/payload#9490] [Q&A] [answered]
After upgrading to the v3 stable version from beta I'm getting this error at startup: `Error: ServerFunctionsProvider requires a serverFunction prop`.

Does anyone know what might cause this?

**Comment by r1tsuu** [comment, marked as answer]
Hey @mattiaz9, this is likely due to an outdated version of the `app/(payload)` folder and specifically, this file - https://github.com/payloadcms/payload/blob/main/templates/website/src/app/(payload)/layout.tsx

I'd suggest you to just copy this folder from the blank/website template to your project.

**Reply by mattiaz9** [reply]
thank you, that was it!
